<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sub_category_input</class>
 <widget class="QMainWindow" name="sub_category_input">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1470</width>
    <height>892</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>861</width>
      <height>541</height>
     </rect>
    </property>
    <property name="title">
     <string>Transactions</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>23</x>
       <y>72</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Category</string>
     </property>
    </widget>
    <widget class="QComboBox" name="category_input">
     <property name="geometry">
      <rect>
       <x>93</x>
       <y>70</y>
       <width>256</width>
       <height>26</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Expense</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Revenue</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="view_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>116</y>
       <width>94</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>View</string>
     </property>
    </widget>
    <widget class="QPushButton" name="add_button">
     <property name="geometry">
      <rect>
       <x>109</x>
       <y>116</y>
       <width>94</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QPushButton" name="delete_button">
     <property name="geometry">
      <rect>
       <x>209</x>
       <y>116</y>
       <width>94</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
    <widget class="QPushButton" name="search_button">
     <property name="geometry">
      <rect>
       <x>739</x>
       <y>116</y>
       <width>94</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="transactions_table">
     <property name="geometry">
      <rect>
       <x>16</x>
       <y>153</y>
       <width>831</width>
       <height>321</height>
      </rect>
     </property>
     <property name="rowCount">
      <number>0</number>
     </property>
     <property name="columnCount">
      <number>8</number>
     </property>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Category</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Date</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Payment Method</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Sub Category</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Account Number</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Amount</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Expense Type</string>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>366</x>
       <y>72</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Sub-Category</string>
     </property>
    </widget>
    <widget class="QComboBox" name="sub_category_input_2">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>70</y>
       <width>278</width>
       <height>26</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Purchase</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sales</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Receipt</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Payment</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="reset_button">
     <property name="geometry">
      <rect>
       <x>637</x>
       <y>116</y>
       <width>94</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
    <widget class="QPushButton" name="close_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>480</y>
       <width>94</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1470</width>
     <height>36</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
